src/engines/import_data_engine.py

```diff
@@ -4,8 +4,7 @@
 from data_import import CSVReader, JSONReader, DatabaseReader
 from database import DatabaseObjectManager, Writer
 from state import State
-
-DB_NAME = 'test1'
+from engines import DB_NAME
```

Comment:
_engines_ imports _ImportDataEngine_ so there's a cyclic import error
```suggestion
from engines.config import DB_NAME
```
above fixes it, but goes against our convention